pkgs/applications/science/logic/protege/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchzip
+, jre
+, makeWrapper
+, makeDesktopItem
+, iconConvTools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "protege";
+  version = "5.5.0";
+
+  src = fetchzip {
```

Comment:
I'm not absolutely against using this zip file, but there's a new part of the manual explaining how to build maven projects in nixpkgs from source:

https://nixos.org/manual/nixpkgs/unstable/#maven

Could you please give it a try?

Reply:
Thank you for the review!
I didn't have time to try the maven packaging yet, however @JonathanReeve would like to start using the package using the current solution proposed by @mgttlinger .
Could we maybe merge this version if it's ok and refine upon it on the future?

Reply:
@doronbehar the technique suggested in the manual apparently fails to detect the dependency for org.apache.felix:maven-bundle-plugin. As the translation from pom.xml to project-info.json is not fully clear to me, I also went for a direct JAR package.